src/transactions/RevokeSponsorshipOpFrame.cpp

```diff
@@ -43,6 +43,7 @@ getAccountID(LedgerEntry const& le)
         return le.data.data().accountID;
     case CLAIMABLE_BALANCE:
         return *le.ext.v1().sponsoringID;
+    case SPEEDEX_CONFIGURATION:
```

Comment:
This is a bit unsafe. Our general rule of thumb is that logic errors should throw an exception to rollback the transaction rather than crashing the node, because crashing the node in this case will actually crash the entire network. Honestly, maybe these `default` cases should work like that too. Please don't change the `default` case (I'd want to think about it) but please throw on `SPEEDEX_CONFIGURATION`.

Reply:
Ok, changed the runtime error. Also added LIQUIDITY_POOL to the other places where it doesn't change the behavior for the sake of consistency.